crates/openvino/src/prepostprocess.rs

```diff
@@ -186,17 +183,13 @@ impl PrePostProcess {
 
     /// Retrieves the output information by name.
     pub fn get_output_info_by_name(&self, name: &str) -> Result<PreProcessOutputInfo> {
-        let mut output_info = std::ptr::null_mut();
-        let c_layout_desc = CString::new(name).unwrap();
+        let mut instance = std::ptr::null_mut();
         try_unsafe!(ov_preprocess_prepostprocessor_get_output_info_by_name(
             self.instance,
-            c_layout_desc.as_ptr(),
-            std::ptr::addr_of_mut!(output_info)
+            cstr!(name),
+            std::ptr::addr_of_mut!(instance)
         ))?;
-
-        Ok(PreProcessOutputInfo {
-            instance: output_info,
-        })
+        Ok(PreProcessOutputInfo { instance })
```

Comment:
Replacing `output_info` by `instance` and similar changes seem good for conciseness, would you want to keep it uniform and make similar changes throughout? E.g. `get_output_info_get_tensor_info` and several others.

Reply:
Just wait for the next PR... I am about to rename all of that.